static/js/webworker-around.js

```diff
@@ -1,18 +1,16 @@
 onmessage = _ => {
-  fetch("/static/test.json")
-    .then(response => {
-      return response.json();
-    })
+  fetch('/static/test.json')
```

Comment:
Now that you're a big boy, maybe is it time for you to read directly from the API… ;)
`fetch('http://158.58.170.155:3030/spots')`
This way you can get rid of the _test.json_ file 😌